seed/data_importer/match.py

```diff
@@ -186,7 +213,7 @@ def match_and_link_incoming_properties_and_taxlots(file_pk, progress_key):
     }
 
 
-def filter_duplicate_states(unmatched_states):
+def filter_duplicate_states(unmatched_states, sub_progress_key=None):
```

Comment:
Is this the best way to modify the existing functions? These fns are called in other locations that won't have `sub_progress_key` arguments. It works, but it leads to a lot of `if ...` statements